integration/tests/economy/test_economics.py

```diff
@@ -124,7 +123,12 @@ def test_send_tokens_without_chain_id(
         )
         instruction_tx.pop("chainId")
 
-        w3_client.send_transaction(account_with_all_tokens, instruction_tx)
+        try:
+            w3_client.send_transaction(account_with_all_tokens, instruction_tx)
```

Comment:
let's split if for 2 tests: first one to check economy for transaction without chain_id (only for neon chain) and second move from economy tests to basic with error message check (for some another chain exept neon)

Reply:
Done